src/Raven.Client/Documents/Session/Operations/Lazy/LazySuggestionQueryOperation.cs

```diff
@@ -56,7 +56,6 @@ public void HandleResponse(GetResponse response)
 
         private void HandleResponse(QueryResult queryResult)
         {
-            _invokeAfterQueryExecuted?.Invoke(queryResult);
```

Comment:
When this event is invoked in lazy suggestion query now?

Reply:
in 'ProccessResult'